src/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -331,4 +331,42 @@ public with sharing class AFFL_AccRecordType_TEST {
         system.assertEquals(hh.Id, markUpdated.Primary_Household__c);
         system.assertEquals(null, markUpdated.Primary_Organization__c);
     }
+
+    @isTest
+    public static void duplicateAffiliationCheck() {        
+        setup();
+
+        // Insert a business account
+        Account bizAcc = UTIL_UnitTestData_API.getMultipleTestAccounts(1, orgRecTypeID)[0];
+        insert bizAcc;
+
+        // Insert a contact with business account as Primary business organization
+        Contact contact = UTIL_UnitTestData_API.getContact();
+        insert contact;
+        
+        //Verify default household account has been created for the contact
```

Comment:
@skosgi1 This comment isn't really valid and can be removed, I think -- we're inserting a contact and then setting the account as the Primary business organization (the comment on line 343 applies to what we're doing here)